Makefile

```diff
@@ -1,72 +1,80 @@
+all: test local-build
 
 # Image URL to use all building/pushing image targets
-REPO ?= scylladb/scylla-operator
-TAG ?= $(shell git describe --tags --always)
-IMG ?= $(REPO):$(TAG)
+REPO	?= scylladb/scylla-operator
+TAG		?= $(shell git describe --tags --always)
+IMG		?= $(REPO):$(TAG)
 
 .EXPORT_ALL_VARIABLES:
-DOCKER_BUILDKIT = 1
-GO111MODULE = off
-KUBEBUILDER_ASSETS = $(CURDIR)/bin/deps
-PATH := $(CURDIR)/bin/deps:$(CURDIR)/bin/deps/go/bin:$(PATH)
-GOROOT = $(CURDIR)/bin/deps/go
-GOVERSION = $(shell go version)
-
-all: test local-build
+DOCKER_BUILDKIT		:= 1
+GO111MODULE			:= off
+KUBEBUILDER_ASSETS	:= $(CURDIR)/bin/deps
+PATH				:= $(CURDIR)/bin/deps:$(CURDIR)/bin/deps/go/bin:$(PATH)
+GOROOT				:= $(CURDIR)/bin/deps/go
+GOVERSION			:= $(shell go version)
 
 # Run tests
+.PHONY: test
 test: fmt vet vendor
-	./bin/deps/go/bin/go test ./pkg/... ./cmd/... -coverprofile cover.out
```

Comment:
If we don't use the downloaded Go anymore we should also avoid downloading it.

Reply:
We use it's in path.

Reply:
Yes but there is a script that downloads it to `bin/deps` and that seems wasteful now.

Reply:
It is what it is we can remove it in another PR, this works with both project and local go.

Reply:
Will it work? I am not sure how the github actions do it to run unit tests.

Reply:
It seems to run fine. :)